src/fides/api/models/fides_user_invite.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+from datetime import datetime, timedelta, timezone
+from typing import Any
+
+from citext import CIText
+from sqlalchemy import Column, String
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.orm import Session
+
+from fides.api.cryptography.cryptographic_util import generate_salt, hash_with_salt
+from fides.api.db.base_class import Base
+
+INVITE_CODE_TTL_HOURS = 24
```

Comment:
nit: could move this to a config var with a default

Reply:
Going to cut scope here and just leave it hard-coded for now